app/app_diagrams_generator.rb

```diff
@@ -0,0 +1,172 @@
+require 'yaml'
+
+class AppDiagramGenerator
+  def initialize
+    @repository_path = Dir.getwd
+    @application_data_load_path = File.join(@repository_path, "data/applications.yml")
+    @application_diagrams_save_path = File.join(@repository_path, "diagrams/")
```

Comment:
You'll need to create the directory otherwise the script will file at line 52. The following should do it

```ruby
FileUtils.mkdir_p(File.join(@repository_path, "diagrams/"))
```

Reply:
Thanks, I've added this